### ISSUE TITLE
Improvement possibility 

### ISSUE THREAD

**Issue by aliafzoon** [mazurowski-lab/SegmentAnyBone#9]
Dear SegmentAnyBone team,

I hope you are doing well. We would like to express our interest in improving the existing model. We are a research team based in Bremen, Germany and we are looking to improve the performance of the model on Vertebrae segmentation. We have vertebrate data and intend to finetune the model. We would appreciate it if you could share some material such as training routine, scripts and test set in order to make sure the process goes well and the model does not lose performance on other parts. 

If you are open to this, please let us know at your earliest convenience to discuss fuerther.  I look forward to hearing back from you.

Best regards,
Ali

**Comment by Guhanxue** [comment]
Hi, thanks so much for your interest! Let me discuss with my advisor,  do you mind leaving me an email address that i can reach out to you later.

**Comment by aliafzoon** [comment]
Sure. Here is my email address:
ali.afzoon.khiyavi@mevis.fraunhofer.de
I hope to hear back from you soon.
